File: src/main/java/aurocosh/divinefavor/common/item/ItemBanishingWand.java
package aurocosh.divinefavor.common.item;

import aurocosh.divinefavor.DivineFavor;
import aurocosh.divinefavor.common.constants.ConstMainTabOrder;
import aurocosh.divinefavor.common.item.base.ModItem;

public class ItemBanishingWand extends ModItem {
    public ItemBanishingWand() {
        super("banishing_wand","banishing_wand", ConstMainTabOrder.TOOLS);
        setMaxStackSize(1);
        setCreativeTab(DivineFavor.TAB_MAIN);
    }
}

File: src/main/kotlin/aurocosh/divinefavor/common/item/tools/ItemBanishingWand.kt
package aurocosh.divinefavor.common.item.tools

import aurocosh.divinefavor.DivineFavor
import aurocosh.divinefavor.common.constants.ConstMainTabOrder
import aurocosh.divinefavor.common.item.base.ModItem

class ItemBanishingWand : ModItem("banishing_wand", "banishing_wand", ConstMainTabOrder.TOOLS) {
    init {
        setMaxStackSize(1)
        creativeTab = DivineFavor.TAB_MAIN
    }
}